packages/datagateway-download/cypress/support/commands.js

```diff
@@ -57,14 +92,25 @@ export const readSciGatewayToken = () => {
   };
 };
 
-Cypress.Commands.add('login', (username, password) => {
-  cy.request('POST', `http://scigateway-preprod.esc.rl.ac.uk:5000/sessions`, {
-    username: username,
-    password: password,
+Cypress.Commands.add('login', (username, password, noRootCredentials) => {
+  let credentials = !noRootCredentials
```

Comment:
You can probably just change this command to accept `username` and `password` params and change any root logins to use call it with `cy.login("root", "pw")`